clang/include/clang-c/Dependencies.h

```diff
@@ -222,6 +222,17 @@ clang_experimental_DependencyScannerWorker_getFileDependencies_v1(
     CXDependencyScannerWorker Worker, int argc, const char *const *argv,
     const char *WorkingDirectory, CXModuleDiscoveredCallback *MDC,
     void *Context, CXString *error);
+
+/**
+ * Same as \c clang_experimental_DependencyScannerWorker_getFileDependencies_v1,
+ * but get the dependencies by module name alone.
+ */
+CINDEX_LINKAGE CXFileDependencies *
+clang_experimental_DependencyScannerWorkerByModName_getFileDependencies_v0(
```

Comment:
Since this now doesn't require a special worker, I think `clang_experimental_DependencyScannerWorker_getModuleDependencies_v0` would be a clearer name. WDYT?

Reply:
It's not clear to me whether the function name should end with `getFileDependencies_v0` or `getModuleDependencies_v0`. The return type `CXFileDependencies` contains both `FileDeps` and `ModuleDeps`, so it seems to me that a name ending with `getFileAndModuleDependencies_v0` would be more accurate, but the two functions declared above this one both use `getFileDependencies`.